src/bot.js

```diff
@@ -30,7 +34,22 @@ let initializeBot = () => {
 			const splitMessage = content.split(' ');
 			const command = splitMessage[1];
 			let data = splitMessage[2];
-            
+
+			//TODO: make me less gross
```

Comment:
This chunk of code can be reduced to `splitMessage.slice(2, splitMessage.length)`